lib/api/objectPutCopyPart.js

```diff
@@ -211,8 +212,9 @@ function objectPutCopyPart(authInfo, request, sourceBucket,
             sourceLocationConstraintName, next) {
             const mpuOverviewKey =
                 `overview${splitter}${destObjectKey}${splitter}${uploadId}`;
+            const options = addIsNonversionedBucket({}, destBucketMD);
```

Comment:
The get is on the MPU special bucket here, not on the destination bucket. For internal buckets, no need to pass the option as they are all in v0.